shared/flexible-classroom/quickstart/web.mdx

```diff
@@ -44,19 +44,19 @@ To prepare your development environment:
 
 ### Download the source code
 
-The source code for Flexible Classroom is located in the Github [CloudClass-Desktop](https://github.com/AgoraIO-Community/CloudClass-Desktop) repository. To download the source code, take the following steps:
+The source code for Flexible Classroom is located in the Github [CloudClass-Desktop](https://github.com/AgoraIO-Community/flexible-classroom-desktop.git) repository. To download the source code, take the following steps:
```

Comment:
The new repository is called **flexible-classroom-desktop**, which I assume should be fixed across this entire page, including the name of the folder/project